packages/cli/lib/index.js

```diff
@@ -32,19 +32,18 @@ let prog = sade('preact').version(pkg.version);
 
 prog
 	.command('build [src]')
-	.describe('Create a production build')
+	.describe(
+		'Create a production build. You can disable "default: true" flags by prefixing them with --no-<option>'
+	)
 	.option('--src', 'Specify source directory', 'src')
 	.option('--dest', 'Specify output directory', 'build')
 	.option('--cwd', 'A directory to use instead of $PWD', '.')
+	.option('--esm', 'Builds ES-2015 bundles for your code', true)
 	.option('--sw', 'Generate and attach a Service Worker', true)
+	.option('--babelConfig', 'Path to custom Babel config', '.babelrc')
 	.option('--json', 'Generate build stats for bundle analysis')
 	.option('--template', 'Path to custom HTML template')
 	.option('--preload', 'Adds preload tags to the document its assets', false)
-	.option(
-		'--refresh',
-		'Enables experimental preact-refresh functionality',
-		false
-	)
```

Comment:
Unless I'm mistaken, this was a mistake? I believe this should go under the `watch` command so I moved it there.